tests/test_validate_parameters.py

```diff
@@ -61,6 +62,15 @@ def get_loaded_yaml(step_spec, input_spec):
             "Step is missing required parameter 'xoutputfile'" in exc_info.value.args[0]
         )
 
+    # Dangerous operation used in parameter
+    with runner.isolated_filesystem():
+        reana_yaml = get_loaded_yaml(
+            cwl_workflow_spec_step, cwl_workflow_spec_dangerous_input_param
+        )
+        _validate_cwl_parameters(reana_yaml)
+        captured = capsys.readouterr()
+        assert 'Operation "sudo python" found in "baseCommand"' in captured.out
+
```

Comment:
We probably don't need tests for this. We're already testing the functionality [here](https://github.com/reanahub/reana-client/blob/20dc10301b74766b3f9c27daba6608d1e65a3840/tests/test_validate_parameters.py#L243).